third_party/bitstream/Readme.md

```diff
@@ -0,0 +1,2 @@
+Bitstream reader.
```

Comment:
Do we need this extra library? I was hoping to keep the dependencies for this library down to zero. I guess so long as we can keep it single header which I think we can. If we are using this we might want to pull it in with cmake instead.

Reply:
This is the library from python CAN tools. This are two files, a .c and a .h. I created the cmake file for it to be able to import it to dbc_parser